src/particles.zig

```diff
@@ -18,10 +19,14 @@ const Vec4d = vec.Vec4d;
 const Vec3f = vec.Vec3f;
 const Vec4f = vec.Vec4f;
 const Vec3i = vec.Vec3i;
+const Vec2f = vec.Vec2f;
+
+var seed: u64 = undefined;
```

Comment:
I just removed this. Please use the global seed which exists exactly for this purpose.